docs/json/radarr/cf/french-voq.json

```diff
@@ -0,0 +1,17 @@
+{
+  "trash_id": "f7caa1942be5cc547c266bd3dbc2cda9",
```

Comment:
are there supposed to be no scores? if so, looks good to me 😄

Reply:
Yes, both are informative. It is like all the other French audio CFs. They exist just for people to know which French they will potentially get 😅